addons/hr_holidays/views/hr_leave_type_views.xml

```diff
@@ -123,7 +88,7 @@
         <field name="type">ir.actions.act_window</field>
         <field name="res_model">hr.leave.type</field>
         <field name="view_type">form</field>
-        <field name="view_mode">tree,kanban,form</field>
+        <field name="view_mode">tree,form</field>
```

Comment:
I don't fully agree with the removal of the kanban view as it may be used in mobile as "mobile list view". Maybe we could keep a small kanban view with the mobile class, like we already do for other models.